logical/system_view.go

```diff
@@ -6,12 +6,12 @@ import "time"
 // for logical backends to consume
 type SystemView interface {
 	// DefaultLeaseTTL returns the default lease TTL set in Vault configuration
-	DefaultLeaseTTL() time.Duration
+	DefaultLeaseTTL() (time.Duration, error)
```

Comment:
We should just ensure the error is not possible, I don't really see how it could happen given we are referencing values we have in-memory.